package.json

```diff
@@ -22,11 +22,11 @@
     "i18n:compile": "lingui compile",
     "i18n:extract": "lingui extract",
     "lint": "eslint --ext ts,tsx src",
-    "start": "vite --mode production",
+    "start": "vite --mode production --host",
     "start-adpr": "vite --mode adpr --host",
-    "start-dev": "vite --mode dev",
-    "start-stg": "vite --mode stg",
-    "start-prod": "vite --mode production",
+    "start-dev": "vite --mode dev --host",
+    "start-stg": "vite --mode stg --host",
+    "start-prod": "vite --mode production --host",
```

Comment:
revert

Reply:
@viet-nv told me to keep these

Reply:
revert pls, mắc công whitelist login url

Reply:
local mà @nguyenhoaidanh. sao lại whitelist

Reply:
cái này auto expose host để debug local vào bằng đt cho tiện